resources/warnings.json

```
@@ -16991,7 +16991,7 @@
     "versions": [
       {
         "lastVersion": "2.5",
-        "pattern": ".*"
+        "pattern": "(1|2[.][0-5])(|[.-].+)"
```

Same question as https://github.com/jenkins-infra/update-center2/pull/780/files#r1598320040
```suggestion
        "pattern": "(1[.]|2[.][0-5])(|[.-].+)"
```